### DISCUSSION TITLE
How to remove automatic numbering from nodes

### DISCUSSION THREAD

**Discussion by bnpwalter** [freeplane/freeplane#1862] [Help]
I've created a template.  Set the View Settings to Outline View.  Whenever I create a new child node the node inserts a sequence number automatically.  
How to eliminate the auto-numbering?
![Forum_Remove-Autonumbering](https://github.com/freeplane/freeplane/assets/50636483/dcc50777-9be6-4de3-a1c5-5401422d92b4)

Also, when exporting to Libreoffice (.odt) the result shows numbering.  
How does one prevent numbering in LO?

![Forum_Libreoffice Outline numbering](https://github.com/freeplane/freeplane/assets/50636483/4ef46796-f62c-4a13-9397-e25788dc577b)

**Comment by euu2021** [comment]
go into `Format->Manage Styles->Edit styles`

Select the style, and disable the numbering:

![image](https://github.com/freeplane/freeplane/assets/77707706/545788e4-93ab-498b-886b-17773026d42d)
